packages/integration/test/integration.spec.ts

```diff
@@ -251,4 +253,47 @@ describe("Integration", () => {
     const fulfillEvent = await fulfillEventPromise;
     expect(fulfillEvent).to.be.ok;
   });
+
+  it.skip("should send Native tokens", async function () {
```

Comment:
Why is this skipped?

Reply:
I was doing some changes and wanted to check the before config.